.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: Bug
```

Comment:
Does it matter if it's lower or upper case? We have a `bug` label, so I just want to make sure that it'll be the same one used

Reply:
updated